openedx_events/content_authoring/data.py

```diff
@@ -180,3 +180,15 @@ class LibraryBlockData:
 
     library_key = attr.ib(type=LibraryLocatorV2)
     usage_key = attr.ib(type=LibraryUsageLocatorV2)
+
+
+@attr.s(frozen=True)
+class ContentObjectData:
+    """
+    Data about content object where tags changed
```

Comment:
The docstring says _where tags changed_ but the data attributes themselves are not associated with tagging, so could this be simply a data representation of the content?

Reply:
Agreed, I changed it to be more generic, representing the content object.